docker/iobio-stack.yml

```diff
@@ -31,7 +31,7 @@ services:
     networks:
       - proxy
     volumes:
-        - ../sqlite/data:/iobio-gru-backend/data
+        - ../sqlite/data:/data
```

Comment:
This was bind mount was causing the problem with annotating variants.  Gru 1.6.1 has some hardcoded paths that expect the files to be located at `/data`, whereas before, any arbitrary directory would have worked.